webassembly-language-support.md

```diff
@@ -23,7 +23,7 @@ Some languages, like CSS, PowerShell, and "Shell", don't really have a meaningfu
 | [JavaScript][JavaScript]  | ⏳  | ⏳ | ⏳ | |
 | [Python][Python]          | ⏳ | ✅  | ✅  |  |
 | [Java][Java]              |  ✅  | ✅  | ❌ ||
-| [PHP][PHP]                | ✅ | ❌ | ❌ ||
+| [PHP][PHP]                | ✅ | ? | ✅ ||
```

Comment:
I wasn't sure about the meaning of `Other` in this context.

Reply:
Seems like the "Other" column refers to non-browser environments that don't support the WASI standard.  Some of the later tables have a column called `CLI` in place of this.  @technosophos can you expand on what was intended here?

Reply:
Originally, it was a catch-all for CLI, embedded environment (e.g. Envoy), etc. I need to just rewrite this table and remove that column, since it rarely tells anything interesting.

Reply:
So since PHP compiled as CGI can be run from the `wasmtime` CLI, we should probably go with ✅